### DISCUSSION TITLE
How to change font?

### DISCUSSION THREAD

**Discussion by leoplct** [thuliteio/doks#492] [General]
I've edited this file to replace Jost to Roboto but it keeps using Jost. 
Fonts file are saved here: static/fonts/vendor/Roboto/Roboto-Regular.ttf and I can successfully download it from http://localhost:1313/fonts/vendor/Roboto/Roboto-Regular.ttf

I've run
```
npm run clean   
npm run server
```

**assets/scss/common/_fonts.scss**
```
/* jost-regular - latin */
@font-face {
  font-family: "Roboto";
  font-style: normal;
  font-weight: 400;
  font-display: swap;
  src:
    local("Roboto"),
    url("fonts/vendor/Roboto/Roboto-Regular.ttf") format("ttf");
}

/* jost-500 - latin */
@font-face {
  font-family: "Roboto";
  font-style: normal;
  font-weight: 500;
  font-display: swap;
  src:
    local("Roboto"),
    url("fonts/vendor/Roboto/Roboto-Medium.ttf") format("ttf");
}

/* jost-700 - latin */
@font-face {
  font-family: "Roboto";
  font-style: normal;
  font-weight: 700;
  font-display: swap;
  src:
    local("Roboto"),
    url("fonts/vendor/Roboto/Roboto-Bold.ttf") format("ttf");
}

/* jost-italic - latin */
@font-face {
  font-family: "Roboto";
  font-style: italic;
  font-weight: 400;
  font-display: swap;
  src:
    local("Roboto"),
    url("fonts/vendor/Roboto/Roboto-Italic.ttf") format("ttf");
}

/* jost-500italic - latin */
@font-face {
  font-family: "Roboto";
  font-style: italic;
  font-weight: 500;
  font-display: swap;
  src:
    local("Roboto"),
    url("fonts/vendor/Roboto/Roboto-MediumItalic.ttf") format("ttf");
}

/* jost-700italic - latin */
@font-face {
  font-family: "Roboto";
  font-style: italic;
  font-weight: 700;
  font-display: swap;
  src: local("Roboto"), url("fonts/vendor/Roboto/Roboto-BoldItalic.ttf") format("ttf");
}
...
```

**Comment by h-enk** [comment]
Will get back to you (bit busy). See also: https://github.com/h-enk/getdoks.org/issues/183

**Reply by hauni97** [reply]
Would be great to know how to change the font.

**Reply by h-enk** [reply]
Totally agree. I will write down a little guide. See also: https://getdoks.org/showcase/functionalities/custom-font/

**Reply by h-enk** [reply]
https://github.com/h-enk/doks/issues/559

**Reply by ImaCrea** [reply]
I've added a solution in the issue linked above, in case it might help ;)

**Comment by dxsup** [comment]
I had the same problem as you, and I finally figured out why this didn't work. I have successfully changed the font after changing the `tff` format to `woff/woff2`. I’m entirely new to web development, so I don't know why only the format woff/woff2 works here.

**Reply by h-enk** [reply]
See: https://github.com/h-enk/getdoks.org/issues/219#issuecomment-1308340223
